docs/source-2.0/implementations.rst

```diff
@@ -75,6 +75,11 @@ IDE Support
       - Java
       - WIP
       - A Language Server Protocol implementation for the Smithy IDL.
+    * - `Tree Sitter grammar for Smithy <https://github.com/indoorvivants/tree-sitter-smithy>`_
+      - C
```

Comment:
nit: the tree sitter grammar is written in JS, but the output parser is in C.

Maybe we should list both JS / C?